src/App.js

```diff
@@ -1,10 +1,14 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```

Comment:
- [ ] Please, kindly update your readme to use the [Microverse template](https://github.com/microverseinc/readme-template) with all its sections (except for tests and live demo if not required).
- [ ] OPTIONAL: I suggest you remove all the text, styles, and images added by the CRA that are no longer needed.

That way, your project will be well documented and cleaner💪

Reply:
thanks for the feedback